ansible/tasks/delete_deployment.yaml

```diff
@@ -0,0 +1,8 @@
+---  
+  - name: Delete {{ react_infra_name }} of PR {{ PR }} in {{ namespace }} namespace
+    shell: >
+      oc delete -n "{{ namespace }}" all --selector pr={{ PR }}
+  - debug: msg="Attempt to delete PR deployment failed"
```

Comment:
Should we also delete the image stream?

Reply:
Just saw the code below

Reply:
I wasn't sure about adding that, but I'm glad you agree about it.